### PR TITLE
Fixes #765: Update shields off description

### LINKED ISSUE #765: Update Shields Off description to match desktop
## Description: 
Shields off description only covers ads and trackers:

screen shot 2019-01-19 at 12 24 27 pm

Shields off description on desktop browser:
shields off

## Expected Result
Update the description to match desktop string:
Browser - shields off

### Steps to Reproduce 
  1. Navigate to any site
  2. Turn shields off

**Reproduces how often:** [Easily reproduced, Intermittent Issue]
Easy

**Brave Version:** 
Dev

**Device details:** 
iPhone SE

### PR DESCRIPTION
**Pull request by rossmoody**
Closes https://github.com/brave/brave-ios/issues/765: Update shields off description

I saw this and reached out to Karen about updating the string to match desktop. Instead of dynamically changing the title, I added the snippet about shields being down.

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-ios/issues) for my issue if one did not already exist.
- [x] My patch or PR title has a standard commit message that looks like `Fix #123: This fixes the shattered coffee cup!` (or `No Bug: <message>` if no relevant ticket)
- [x] *Unit Tests* are updated to cover new or changed functionality
- [ ] User-facing strings use `NSLocalizableString()`
- [x] New files have MPL-2.0 license header.

## Test Plan:

1. Turn shields off.
2. Make sure descriptions matches `Brave Shields are currently down for this site. You\'re browsing without any privacy and security protections.`

### Screenshots:

image

## Reviewer Checklist:

- [ ] PR is linked to an issue via [Zenhub](https://www.zenhub.com/extension).
- [ ] Issues are assigned to at least one epic.
- [ ] Issues include necessary QA labels:
  - [ ] `QA/(Yes|No)`
  - [ ] `release-notes/(include|exclude)`
  - [ ] `bug` / `enhancement`
- [ ] Necessary security reviews have taken place.
- [ ] Adequate test coverage exists to prevent regressions.
- [ ] Adequate test plan exists for QA to validate (if applicable)

### CODE REVIEW

File: BraveShared/BraveStrings.swift [hunk 1]
```diff
@@ -475,8 +475,7 @@ extension Strings {
...
     public static let Fingerprinting_Protection_wrapped = NSLocalizedString("FingerprintingnProtection", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Fingerprinting\nProtection", comment: "blocking stat title")
-    public static let Shields_Overview = NSLocalizedString("ShieldsOverview", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Site Shields allow you to control when ads and trackers are blocked for each site that you visit. If you prefer to see ads on a specific site, you can enable them here.", comment: "shields overview message")
-    public static let Shields_Overview_Footer = NSLocalizedString("ShieldsOverviewFooter", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Note: Some sites may require scripts to work properly so this shield is turned off by default.", comment: "shields overview footer message")
+    public static let Shields_Overview = NSLocalizedString("ShieldsOverview", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Brave Shields are currently down for this site. You\'re browsing without any privacy and security protections.", comment: "shields overview message")
```
Comment (jhreis) [inline BraveShared/BraveStrings.swift]: Escaping is not needed here for the `'`, please remove.

**Review by jhreis — CHANGES_REQUESTED** [review]

**Comment by jhreis** [comment]
@tomlowenthal can you please review this copy change. 

Two main changes:
- Completely removing:
> Note: Some sites may require scripts to work properly so this shield is turned off by default.

- Modifying
> Site Shields allow you to control when ads and trackers are blocked for each site that you visit. If you prefer to see ads on a specific site, you can enable them here.

to 

> Brave Shields are currently down for this site. You're browsing without any privacy and security protections.

**Comment by jhreis** [comment]
@rossmoody one of the issues with the new copy is that this is also shown on the new tab page, which isn't technically insecure, and there is now way to toggle shields on for new tab page.

**Comment by rossmoody** [comment]
Ah, I see. Well, I'll let @karenkliu or @tomlowenthal chime in on correct direction, I was just trying to help out but it seems like it might require more functionality updates. Could always consider shutting this one down in favor of more specific spec.

**Comment by karenkliu** [comment]
If we're aiming for desktop parity, then Shields should not be interactive on the new tab page or system preferences pages (it would be in a disabled state).

Screen Shot 2019-07-02 at 12 22 36 PM

**Comment by rossmoody** [comment]
Gonna close this one. Needs a different approach to what is going on here.

**Pull request closed without merging**